src/zoid/buttons/component.jsx

```diff
@@ -451,6 +451,12 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                 value:      getMerchantID
             },
 
+            buttonsArray: {
```

Comment:
Can we drop the "Array" part from the prop name? Perhaps "buttons" or "renderedButtons" would work. The other existing props using `type: 'array'` do not include Array in the name so I would prefer not to do that here.

```suggestion
            buttons: {
```

Reply:
agreed with a preference towards renderedButtons

Reply:
Done!